webview-ui/src/components/ChatView.tsx

```diff
@@ -75,14 +75,43 @@ export const ChatView = ({
     );
   };
 
+  const deleteMessageBlock = (
```

Comment:
If `deleteMessageBlock` is only used in a `<Message>` component, should it just be defined in `Message.tsx`? This definitely might be a design pattern or React thing I'm unfamiliar with, but I'm just curious.

Reply:
This needs to know where the message block is located and the `<Message>` component didn't have that being passed as props. I would rather just pass it the function and it not have any knowledge of where it was located, than pass it where it is located and have it use that. I don't know if it's the best way to do it. It's just the way that I came up with.
The other solution I thought of was giving every message block a unique id and setting up an endpoint on the extension side which you would send the id and the extension side would handle the logic. This seemed like a much less intrusive solution.

Reply:
Ah yep that makes sense!